app/models/share.rb

```diff
@@ -1,6 +1,9 @@
 class Share < ApplicationRecord
   belongs_to :link
 
+  UTM_SOURCES = %w[a b]
+  UTM_MEDIUMS = %w[c d]
```

Comment:
@JuanVqz If you don't know what to put here, you can check out our UTM spreadsheet.

Reply:
Hey good suggestion, we grabbed some from the spreadsheet, thanks!